app/assets/javascripts/discourse/tests/integration/components/site-header-test.js

```diff
@@ -37,16 +37,6 @@ module("Integration | Component | site-header", function (hooks) {
     assert.strictEqual(unreadBadge.textContent, "5");
   });
 
-  test("hamburger menu icon shows pending reviewables count", async function (assert) {
```

Comment:
No need for this test anymore since the legacy navigation menu no longer exists.